log4j-changelog/src/main/resources/log4j-changelog.xsd

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements. See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License. You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<schema xmlns="http://www.w3.org/2001/XMLSchema"
+  targetNamespace="https://logging.apache.org/log4j/log4j-changelog"
+  xmlns:cl="https://logging.apache.org/log4j/log4j-changelog" elementFormDefault="qualified">
```

Comment:
We better have some versioning:

```suggestion
                 targetNamespace="https://logging.apache.org/log4j/changelog/1.0.0"
                 xmlns:cl="https://logging.apache.org/log4j/changelog/1.0.0"
                 elementFormDefault="qualified">
```

Reply:
I would rather keep the namespace constant, but add a `version` attribute (with a single possible value). Like `web-app-versionType` in the Servlet API: https://jakarta.ee/xml/ns/jakartaee/web-common_5_0.xsd